spec/unit/plugins/linux/systemd_paths_spec.rb

```diff
@@ -16,7 +16,7 @@
 # limitations under the License.
 #
 
-require File.expand_path(File.dirname(__FILE__) + "/../../../spec_helper.rb")
+require File.expand_path(__dir__ + "/../../../spec_helper.rb")
```

Comment:
```suggestion
require "spec_helper.rb"
```

`spec/` should be on the path.